src/command_modules/azure-cli-backup/HISTORY.rst

```diff
@@ -0,0 +1,9 @@
+.. :changelog:
+
+Release History
+===============
+
+0.0.1b1 (YYYY-MM-DD)
```

Comment:
Version number should not have the `b1` suffix. It can cause issues with older versions of pip.
Also, instead of `YYYY-MM-DD` use `unreleased`